torchvision/transforms/functional.py

```diff
@@ -1173,3 +1173,118 @@ def gaussian_blur(img: Tensor, kernel_size: List[int], sigma: Optional[List[floa
     if not isinstance(img, torch.Tensor):
         output = to_pil_image(output)
     return output
+
+
+def invert(img: Tensor) -> Tensor:
+    """Invert the colors of a PIL Image or torch Tensor.
```

Comment:
Are we expecting input as RGB images here and for other color ops. Maybe worth to mention that ?

Reply:
I checked and invert can handle other types such as grayscale and makes not assumption over the image being RGB. That's why I don't mention anything about that in the doc.

Reply:
Could be good to at least mention that it expects RGB or grayscale, as the code won't work for other color formats such as YUV (which we might add in the future for better mobile compatibility)

Reply:
Done!